tutorials/models/1_gnn/8_sse_mx.py

```diff
@@ -417,32 +418,62 @@ def train(g, label_nodes, steady_state_operator, predictor, trainer):
 # at a time with neighbor sampling.
 #
 # The following code demonstrates how to use the ``NeighborSampler`` to
-# sample subgraphs, and stores the nodes and edges of the subgraph, as
-# well as seed nodes in each iteration:
+# sample subgraphs, and stores the seed nodes of the subgraph in each iteration:
 #
 nx_G = nx.erdos_renyi_graph(36, 0.06)
 G = dgl.DGLGraph(nx_G.to_directed(), readonly=True)
 sampler = dgl.contrib.sampling.NeighborSampler(
        G, 2, 3, num_hops=2, shuffle=True)
-nid = []
-eid = []
+seeds = []
 for subg, aux_info in sampler:
-    nid.append(subg.parent_nid.asnumpy())
-    eid.append(subg.parent_eid.asnumpy())
+    seeds.append(subg.layer_parent_nid(-1))
 
 ##############################################################################
 # Sampler with DGL
 # ~~~~~~~~~~~~~~~~
 #
 # The code illustrates the training process in mini-batches.
 #
-def update_embeddings_subgraph(g, seed_nodes, steady_state_operator):
+
+class SubgraphSteadyStateOperator(gluon.Block):
+    def __init__(self, n_hidden, activation, **kwargs):
+        super(SubgraphSteadyStateOperator, self).__init__(**kwargs)
+        with self.name_scope():
+            self.dense1 = gluon.nn.Dense(n_hidden, activation=activation)
+            self.dense2 = gluon.nn.Dense(n_hidden)
+
+    def forward(self, subg):
+        def message_func(edges):
+            x = edges.src['x']
+            h = edges.src['h']
+            return {'m' : mx.nd.concat(x, h, dim=1)}
+
+        def reduce_func(nodes):
+            m = mx.nd.sum(nodes.mailbox['m'], axis=1)
+            z = mx.nd.concat(nodes.data['x'], m, dim=1)
+            return {'h' : self.dense2(self.dense1(z))}
+
+        subg.block_compute(0, message_func, reduce_func)
```

Comment:
I realized that we are using NodeFlow interfaces here.  Should we move the tutorial after the NodeFlow one?  Especially since it's a brand new concept and we are not talking (and probably don't want to talk) about NodeFlow here?

Also it seems necessary to adapt the text a little bit, since we are technically not "computing on the subgraph".

Reply:
you are right. i need to fix the tutorial to pass the CI test.